kubernetes.go

```diff
@@ -54,7 +54,7 @@ type Kubernetes struct {
 	InstalledApplications []KubernetesApplications `json:"installed_applications"`
 }
 
-type KubernetesConfig struct {
+type KubernetesClusterConfig struct {
 	Name              string `form:"name"`
 	NumTargetNodes    string `form:"num_target_nodes"`
```

Comment:
This should be an integer though, right?

Reply:
```suggestion
	NumTargetNodes    int `form:"num_target_nodes"`
```
i believe yes :D